gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 group=io.ballerina.stdlib
 version=2.2.3-SNAPSHOT
-ballerinaLangVersion=2201.0.4-rc1
+ballerinaLangVersion=2201.0.4
```

Comment:
Unless we are using some new feature, I think we do not need to update the lang version here. @niveathika WDYT?